docs/book/src/cronjob-tutorial/testdata/project/test/utils/utils.go

```diff
@@ -75,6 +75,18 @@ func UninstallPrometheusOperator() {
 	}
 }
 
+// IsPrometheusOperatorInstalled checks if the Prometheus Operator is
+// installed by verifying the availability of a key deployment.
+func IsPrometheusOperatorInstalled() bool {
+	deployment := "prometheus-operator"
+	cmd := exec.Command("kubectl", "wait", "deployment.apps/"+deployment,
+		"--for", "condition=Available",
+		"--timeout", "10s",
+	)
+	_, err := Run(cmd)
+	return err == nil
+}
```

Comment:
We need to check for the CRDs. If the cluster has Prometheus or CertManager we do not know the name of the deployment, the namespace where it is installed and etc. 

So, I think we need something like

```
func IsPrometheusCRDsInstalled() bool {
	// Execute the kubectl command to get all CRDs that contain "prometheus"
	cmd := exec.Command("kubectl", "get", "crds")
	output, err := Run(cmd)
	if err != nil {
		fmt.Printf("Failed to list CRDs: %v\n", err)
		return false
	}

	// Check if any of the CRDs contain "prometheus"
	crdList := GetNonEmptyLines(string(output))
	for _, crd := range crdList {
		if strings.Contains(crd, "prometheus") {
			fmt.Println("Found a Prometheus-related CRD.")
			return true
		}
	}

	fmt.Println("No Prometheus-related CRDs found.")
	return false
}
```

Reply:
WDYT?

Reply:
Good point. Checking for CRDs would be more reliable since the deployment names and namespaces can vary.